src/test/scala/org/broadinstitute/dsde/firecloud/service/MethodsServiceACLSpec.scala

```diff
@@ -0,0 +1,148 @@
+package org.broadinstitute.dsde.firecloud.service
+
+import org.broadinstitute.dsde.firecloud.mock.MockAgoraACLData._
+import org.broadinstitute.dsde.firecloud.mock.MockAgoraACLServer
+import org.broadinstitute.dsde.firecloud.model.MethodRepository.FireCloudPermission
+import spray.http.HttpMethods
+import spray.http.StatusCodes._
+
+import spray.json._
+import spray.httpx.SprayJsonSupport._
+import spray.json.DefaultJsonProtocol._
+import org.broadinstitute.dsde.firecloud.model.ModelJsonProtocol.impFireCloudPermission
+
+
+class MethodsServiceACLSpec extends ServiceSpec with MethodsService {
+
+  def actorRefFactory = system
+
+  override def beforeAll(): Unit = {
+    MockAgoraACLServer.startACLServer()
+  }
+
+  override def afterAll(): Unit = {
+    MockAgoraACLServer.stopACLServer()
+  }
+
+  val localConfigPermissionsPath = localConfigsPath + "/ns/n/1/permissions"
+  val localMethodPermissionsPath = localMethodsPath + "/ns/n/1/permissions"
+
+  // we have to manually create this faulty json; we can't create it via FireCloudPermission objects, because
+  // they don't allow faulty values!
+  val sourceBadRole = """[{"user":"foo@broadinstitute.org","role":"OWNER"},{"user":"bar@broadinstitute.org","role":"UNKNOWN"}]"""
+  val jsonBadRole = sourceBadRole.parseJson.asInstanceOf[JsArray]
+
+  val sourceBadUser = """[{"user":"foo@broadinstitute.org","role":"OWNER"},{"user":"","role":"READER"}]"""
+  val jsonBadUser = sourceBadUser.parseJson.asInstanceOf[JsArray]
+
+    /* ACL endpoints.
+       We unit test the individual translations, so we only need to augment the unit tests here. We need to test:
+        * handling of lists (unit tests work with single objects)
+        * configs vs. methods paths are not mixed up (we use a single code path for both)
+        * rejections when invalid data POSTed
+        * DELETE/PUT methods are rejected (we supersede those with POST, and should block passthroughs)
+     */
```

Comment:
In light of this approach (which I really like, i.e. splitting out orch logic testing from passthroughs) - we might want to start packaging our tests a little more granularly. Having everything in org.broadinstitute.dsde.firecloud.mock will become difficult to sort through over time.